includes/admin/class-give-settings.php

```diff
@@ -543,6 +543,19 @@ public function give_settings( $active_tab ) {
 							'type' => 'file'
 						),
 						array(
+						    'id'      => 'email_logo_size',

```

Comment:
@philwp nowadays emails are responsive, so I did not think to give the admin the option to select logo size has more benefits than giving admin an option for email logo, so admin has full freedom to select upload their own size.